assets/javascripts/kitten/components/meters/spinner/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import styled, { keyframes } from 'styled-components'
+import classNames from 'classnames'
+import COLORS from '../../../constants/colors-config'
+import { pxToRem } from '../../../helpers/utils/typography'
+
+const circleRadius = 30
+const dashLength = `calc(2 * 3.141592 * ${pxToRem(circleRadius)})`
+
+const rotateAnimate = keyframes`
+  from {
+    stroke-dashoffset: ${dashLength};
+  }
+  to {
+    stroke-dashoffset: calc(${dashLength} * ${({ progressValue }) =>
+  100 - progressValue} / 100);
+  }
+`
+
+const StyledCircle = styled(({ progressColor, progressValue, ...props }) => (
+  <circle {...props} />
+))`
+  fill: transparent;
+  stroke-width: ${pxToRem(4)};
+  stroke-linecap: square;
```

Comment:
C'est pour une plus grande exactitude dans l'affichage des bouts de la ligne


```suggestion
  stroke-linecap: butt;
```

Reply:
🍑